pkg/resourceinterpreter/customizedinterpreter/customized.go

```diff
@@ -251,6 +251,10 @@ func (e *CustomizedInterpreter) callHook(ctx context.Context, hook configmanager
 
 // applyPatch uses patchType mode to patch object.
 func applyPatch(object *unstructured.Unstructured, patch []byte, patchType configv1alpha1.PatchType) (*unstructured.Unstructured, error) {
+	if len(patch) == 0 && len(patchType) == 0 {
+		klog.Infof(" (%s: %s), patch and patchType is nil, give up apply patch ", object.GroupVersionKind().String(), object.GetName())
+		return object, nil
+	}
```

Comment:
```diff
diff --git a/pkg/resourceinterpreter/customizedinterpreter/customized.go b/pkg/resourceinterpreter/customizedinterpreter/customized.go
index 35af4a5b..05e9aed2 100644
--- a/pkg/resourceinterpreter/customizedinterpreter/customized.go
+++ b/pkg/resourceinterpreter/customizedinterpreter/customized.go
@@ -252,7 +252,7 @@ func (e *CustomizedInterpreter) callHook(ctx context.Context, hook configmanager
 // applyPatch uses patchType mode to patch object.
 func applyPatch(object *unstructured.Unstructured, patch []byte, patchType configv1alpha1.PatchType) (*unstructured.Unstructured, error) {
        if len(patch) == 0 && len(patchType) == 0 {
-               klog.Infof(" (%s: %s), patch and patchType is nil, give up apply patch ", object.GroupVersionKind().String(), object.GetName())
+               klog.Infof("Skip apply patch for object(%s: %s) as patch and patchType is nil", object.GroupVersionKind().String(), object.GetName())
                return object, nil
        }
        switch patchType {

```
I think `skip` would be better than `give up`

Reply:
> I think skip would be better than give up

good suggestion